### PR TITLE
added the frist style

### PR DESCRIPTION
**Pull request by Mov305**
Added the UI 
Tested the UI 
Checked the linters

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,20 +1,10 @@
...
-          and save to reload.
-        </p>
-        <a className="App-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
+    <div className="App ">
+      <Calculator />
     </div>
   );
```
Comment (ShahierNashaat) [inline src/App.js]: - [ ] Kindly notice that the `App.js` component should be class-based.

```javascript
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Mov305, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Mov305) [reply src/App.js]: React uses Functional components by default I didn't write it, The components I made were all class

Comment (ShahierNashaat) [reply src/App.js]: You'r right but you also has to change the `App.js` component and make it class based, as it is in the requirements of the project.

**Review by BertrandConxy — APPROVED** [review]
Hi @Mov305 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 

- No linter errors
- project requirements are met
- project works fine


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**